src/content/docs/en/guides/integrations-guide/cloudflare.mdx

```diff
@@ -256,7 +256,7 @@ const { env } = Astro.locals.runtime;
 
 You can access the runtime from API endpoints through `context.locals`:
 
-```ts title="src/pages/api/someFile.js"
+```ts title="src/pages/api/someFile.ts"
```

Comment:
> To create a custom endpoint, add a .js or .ts file to the /pages directory. The .js or .ts extension will be removed during the build process [...]

Astro docs state that endpoints can be `.js` or `.ts` files. So why do we need this change?

Reply:
The code block is specified as `ts`, but the file name is `.js`, so I thought it would be inconsistent and should be fixed.

As you said, if this is what you intended, I'll close this PR :)

Reply:
```suggestion
```js title="src/pages/api/someFile.js"
```

Oh sorry I missed that, are you happy with this suggestion?

Reply:
My thinking was that this file didn't *look* explicitly like a ts file. I would instead have changed the snippet language to js!

I'm happy for it to be either one, but this looks like just js to me, so that would be my preference. We usually default to "people who want to chose ts instead will do that on their own". BUT, if you have reason to believe that e.g. 90% of these adapter users WOULD choose ts instead, I don't mind showing one.

I will leave the final call to @alexanderniebuhr !

Reply:
> Oh sorry I missed that, are you happy with this suggestion?

Sounds good!